client/modules/state/state.go

```diff
@@ -113,35 +80,20 @@ func (s *LevelDBState) NewStateFromOld(stateDbPath string) (State, string, error
 	return state, stateDbPath, err
 }
 
-func (s *LevelDBState) initJsonKey(key []byte, data interface{}) error {
-	if _, err := s.stateDb.Get(key, nil); err != nil {
-		operationsBz, err := json.Marshal(data)
-		if err != nil {
-			return fmt.Errorf("failed to marshal storage structure: %w", err)
-		}
-		err = s.stateDb.Put([]byte(key), operationsBz, nil)
-		if err != nil {
-			return fmt.Errorf("failed to init state: %w", err)
-		}
-	}
-
-	return nil
-}
-
 // Reset creates new underlying leveldb storage
 func (s *LevelDBState) Reset(stateDbPath string) (string, error) {
 	s.Lock()
 	defer s.Unlock()
 
-	if stateDbPath == "" {
+	if len(stateDbPath) < 1 {
```

Comment:
I've seen this before in another PR. Is there a secret logic in checking if a string is empty by checking its len, not by if str == "" ? as for me, this is misleading, cause when I see a len check, I expect a slice or a map, not a string

Reply:
i don't know, i made this branch from fsmservice changes